usr/src/test/zfs-tests/tests/functional/cli_user/misc/zfs_get_001_neg.ksh

```diff
@@ -54,7 +54,7 @@ while [[ $i -lt ${#args[*]} ]]
 do
 	PROP=${props[$i]}
 	EXPECTED=${prop_vals[$i]}
-	ACTUAL=$( zfs get $PROP -o value -H snapdir $TESTPOOl/$TESTFS/prop )
+	ACTUAL=$( zfs get $PROP -o value -H snapdir $TESTPOOL/$TESTFS/prop )
```

Comment:
In trying to figure out how this ever passed, I noticed another bug. On line 53, we're looking at $args, which is undefined so we never execute any of this code.